src/collections/_documentation/platforms/javascript/sourcemaps/troubleshooting.md

```diff
@@ -19,36 +25,102 @@ Double-check that your deployed, final JavaScript files have `sourceMappingURL`
 
 Alternately, instead of `sourceMappingURL`, you can set a `SourceMap` HTTP header on your minified file. If this header is present, Sentry will use it to discover the location of your source map.
 
-### Verify artifact names match sourceMappingURL
+### Verify artifact names match `sourceMappingURL` value
 
-When [uploading source maps to Sentry](#uploading-source-maps-to-sentry), you must name your source map files with the same name found in `sourceMappingURL`.
+The `sourceMappingURL` comment on the last line of your bundled or minified JavaScript file tells Sentry (or the browser) where to locate the corresponding source map. This can either be a fully qualified URL, a relative path, or the filename itself. When uploading artifacts to Sentry, you must name your source map files with the same name found in the  `sourceMappingURL` comment.
 
-For example, if you have the following in a minified application file, `app.min.js`:
+i. e. if your file is similar to:
 
 ```javascript
-//-- end app.min.js
-//# sourceMappingURL=https://example.com/dist/js/app.min.js.map
+    // -- end script.min.js
+    //# sourceMappingURL=script.min.js.map
 ```
 
-Sentry will look for a matching artifact named exactly `https://example.com/dist/js/app.min.js.map`.
+then your uploaded artifact should also be named `script.min.js.map`. 
```

Comment:
This isn't necessarily correct – it all depends on which URL app.min.js (the minified file) resolves to. If it resolves to `http://example.com/js/app.min.js`, then the source map will be located at `http://example.com/js/app.min.js.map` – and the artifact name needs this fully qualified URL (with an asterisk that you can omit host/protocol/port using the `~` shorthand).